nrm_django/grants/admin.py

```diff
@@ -167,6 +178,19 @@ def get_urls(self):
         return my_urls + urls
 
 
+class GrantAuthorityAdmin(admin.ModelAdmin):
+    list_filter = ("authority_cd",)
```

Comment:
Is this a usably short list of things to filter on?

Reply:
97. So it's right on the bubble, and we may want to move it into a select. But it *is* really useful for quickly scanning them.